install.bat

```diff
@@ -617,6 +624,66 @@ local function restore_config_files()
   vars.CONFBACKUPDIR = nil
 end
 
+-- Find GCC based toolchain
+local find_gcc_suite = function()
+
+    -- read output os-command
+    local read_output = function(cmd)
+        local f = io.popen("type NUL && " .. cmd .. ' 2>NUL')
+        if not f then return nil, "failed to open command: " .. tostring(cmd) end
+        local lines = {}
+        while true do
+            local l = f:read()
+            if not l then
+                f:close()
+                return lines
+            end
+            table.insert(lines, l)
+        end
+    end
+    
+    -- returns: full filename, path, filename
+    local find_file = function(mask, path)
+        local cmd
+        if path then
+            cmd = 'where.exe /R "' .. path .. '" ' .. mask
+        else
+            cmd = 'where.exe ' .. mask
+        end
+        local files, err = read_output(cmd)
+        if not files or not files[1] then
+            return nil, "couldn't find '".. mask .. "', " .. (err or "not found")
+        end
+        local path, file = string.match(files[1], "^(.+)%\\([^%\\]+)$")
+        return files[1], path, file
+    end
+
+    local first_one = "*gcc.exe"  -- first file we're assuming to point to the compiler suite
+    local full, path, filename = find_file(first_one, nil)
+    if not full then
+        return nil, path
+    end
+    vars.MINGW_BIN_PATH = path
+
+    local result = {
+        gcc = full
+    }
+    for i, name in ipairs({"make", "ar", "windres", "ranlib"}) do
+        result[name] = find_file(name..".exe", path)
+        if not result[name] then
+            result[name] = find_file("*"..name.."*.exe", path)
+        end
+    end
+
+    vars.MINGW_MAKE = (result.make and '[['..result.make..']]') or "nil  -- not found by intaller"
+    vars.MINGW_CC = (result.gcc and '[['..result.gcc..']]') or "nil  -- not found by intaller"
+    vars.MINGW_RC = (result.windres and '[['..result.windres..']]') or "nil  -- not found by intaller"
+    vars.MINGW_LD = (result.gcc and '[['..result.gcc..']]') or "nil  -- not found by intaller"
+    vars.MINGW_AR = (result.ar and '[['..result.ar..']]') or "nil  -- not found by intaller"
+    vars.MINGW_RANLIB = (result.ranlib and '[['..result.ranlib..']]') or "nil  -- not found by intaller"

```

Comment:
typo in all entries above: `intaller` -> `installer`

Reply:
thx. Fixed.